src/components/table/table.js

```diff
@@ -953,6 +953,13 @@ export default {
       }
       this.$emit('row-clicked', item, index, e)
     },
+    middleMouseRowClicked (e, item, index) {
+      if (this.stopIfBusy(e)) {
+        // If table is busy (via provider) then don't propagate
+        return
+      }
+      this.$emit('row-middle-clicked', item, index, e)
+    },
```

Comment:
Per MDN (https://developer.mozilla.org/en-US/docs/Web/Events/auxclick#Preventing_default_actions):

> When listening for `auxclick` events originating on elements that do not support input or navigation _(i.e. a `<tr>`), you will often want to explicitly prevent other default actions mapped to the down action of the middle mouse button. On Windows this is usually autoscroll, and on macOS and Linux this is usually clipboard paste. This can be done by preventing the default behaviour of the `mousedown` or `pointerdown` event.

So using middle click might have side effects for the user (i.e. a scroll or copy)

The `auxclicked` event is cancelable, but it doesn't cancel the `mousedown` and `mouseup` events that precede it (`auxclicked` happens after `mouseup`).

This may need to be noted in the documentation as well.